_partials/_migrate_post_schema_caggs_etal.md

```diff
@@ -0,0 +1,165 @@
+## Migrate schema post-data
+
+When you have migrated your table and hypertable data, migrate your PostgreSQL
+schema post-data. This includes information about constraints.
+
+<Procedure>
+
+### Migrating schema post-data
+
+1.  At the command prompt, dump the schema post-data from your source database
+    into a `dump_post_data.bak` file, using your source database connection details. Exclude
+    Timescale-specific schemas. If you are prompted for a password, use your
+    source database credentials:
+
+    ```bash
+    pg_dump -U <SOURCE_DB_USERNAME> -W \
+    -h <SOURCE_DB_HOST> -p <SOURCE_DB_PORT> -Fc -v \
+    --section=post-data --exclude-schema="_timescaledb*" \
+    -f dump_post_data.bak <DATABASE_NAME>
+    ```
+
+1.  Restore the dumped schema post-data from the `dump_post_data.bak` file into
+    your Timescale database, using your connection details. To avoid permissions
+    errors, include the `--no-owner` flag:
+
+    ```bash
+    pg_restore -U tsdbadmin -W \
+    -h <HOST> -p <PORT> --no-owner -Fc \
+    -v -d tsdb dump_post_data.bak
+    ```
+
+</Procedure>
+
+### Troubleshooting
+
+If you see these errors during the migration process, you can safely ignore
+them. The migration still occurs successfully.
+
+```bash
+pg_restore: error: could not execute query: ERROR:  relation "<relation_name>" already exists
+```
+
+```bash
+pg_restore: error: could not execute query: ERROR:  trigger "ts_insert_blocker" for relation "<relation_name>" already exists
+```
```

Comment:
Will be highlighted weirdly if you use bash highlights for this.

Reply:
Done.